features/step_definitions/provider_steps.rb

```diff
@@ -124,7 +124,7 @@ def login_form
   GHERKIN
 end
 
-And('As a developer, I login through RH SSO') do
+And('As a developer, I want to login through RH SSO') do
```

Comment:
Why this change? I'd keep "I log in" (well, actually "log in" is two words :grimacing: )

Reply:
I think the step name was wrong because it doesn't login, it only opens the login screen.

Reply:
Ah, OK... 

Then I think to make it more clear this step should be "I see RH-SSO login option on the login page".
But I don't know if it's good either, for sure @josemigallas  wouldn't like it :grimacing:

Reply:
done